dev-util/codeblocks/codeblocks-9999.ebuild

```
@@ -56,6 +56,11 @@ src_configure() {
 		$(use_with contrib contrib-plugins all)
 }
 
+src_install() {
+	default
+	find "${ED}" -name '*.la' -delete || die
```

```suggestion
	find "${ED}" -type f -name '*.la' -delete || die
```
See https://devmanual.gentoo.org/eclass-reference/ltprune.eclass/index.html

---

Done.